pmg/static/resources/javascript/bills.js

```diff
@@ -68,5 +68,6 @@ $(function() {
 
   // show the first tab when the page loads
   var firstTab = $('.bill-version-content a[data-toggle="tab"]')[0];
-  autoShowBillPdf(firstTab.getAttribute('href').slice(1));
+  if (firstTab)
```

Comment:
Reason for this change is the console error that I got that prevented any Javascript defined after `bills.js` to be run.

![Screenshot at 2019-08-21 20-45-23](https://user-images.githubusercontent.com/4767109/63459418-c7ce6500-c454-11e9-9f70-f469843d827d.png)